### PR TITLE
Fix/7704 qamy orcid new functionality for contributors in works not implemented

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA][my-orcid] New functionality for contributors in works not implemented](https://trello.com/c/t8fa5jir/7704-qamy-orcid-new-functionality-for-contributors-in-works-not-implemented)

File: src/app/record/components/work/work.component.html [hunk 1]
```diff
@@ -15,6 +15,28 @@
     <app-display-external-ids
       [externalIdentifiers]="work.workExternalIdentifiers"
     ></app-display-external-ids>
+    <div class="general-data" *ngIf="work?.contributors?.length > 0">
+      <span i18n="#upperCase@@works.contributors"> Contributors </span>:
```
Comment (leomendoza123) [inline src/app/record/components/work/work.component.html:19]: the `upperCase` will make `Contributors` be displayed as  `CONTRIBUTORS` on QA, is this what we want?

Comment (leomendoza123) [reply src/app/record/components/work/work.component.html:19]: We can remove this to avoid it been show in UPPER_CASE

**Review by leomendoza123 — CHANGES_REQUESTED** [review]
Just a small change

File: src/app/record/components/work/work.component.html (hunk 1 above)
Comment (leomendoza123) [reply src/app/record/components/work/work.component.html]: Sorry Dani! Looking at the figma file 
image
We DO need this to be upper case

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**